src/lib/components/constrainedtext/Constrainedtext.component.tsx

```diff
@@ -22,7 +22,6 @@ const ConstrainedTextContainer = styled.div`
   ${(props) =>
     props.lineClamp > 1
       ? `
-  display: -webkit-box;
```

Comment:
Can you describe the use case where the ellipsis was not displayed ? According to mdn docs we can't use webkit-line-clamp without display: -webkit-box or -webkit-inline-box (https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp):

> The -webkit-line-clamp CSS property allows limiting of the contents of a [block](https://developer.mozilla.org/en-US/docs/Glossary/Block) to the specified number of lines.

> It only works in combination with the [display](https://developer.mozilla.org/en-US/docs/Web/CSS/display) property set to -webkit-box or -webkit-inline-box and the [-webkit-box-orient](https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient) property set to vertical.

So bottomline I'm afraid that this change breaks the line-clamp "feature" of the component.

Reply:
The ellipsis was not displayed on any column of Groups or Users table (checked both on Firefox and Chrome). I saw the comment and the documentation, but I think it's outdated

Reply:
Here are some "before and after" screenshots: 
![before-core-ui-ellipsis-fix](https://user-images.githubusercontent.com/106077432/207336913-a9e3515a-ffd5-43f6-ac30-6b63715fe8b5.png)

![after-core-ui-ellipsis-fix](https://user-images.githubusercontent.com/106077432/207336886-dc6c013b-dae2-4328-9fd4-f5376ea27138.png)

Reply:
Okay, I understood the nature of the problem. When `display: -webkit-box;` and `-webkit-line-clamp` are used together, ellipsis are placed only when the content exceeds the given number of lines (in our case 2). When the content is less than the given lines, the behavior is similar to `overflow: hidden` (first screenshot)

Reply:
I'll revert this change. It's less than ideal for shorter text, but we avoid breaking the table for longer one